RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.h

```diff
@@ -0,0 +1,140 @@
+#ifndef PCCNTupler_h
+#define PCCNTupler_h
+
+/** \class PCCNTupler
+ * ----------------------------------------------------------------------
+ * PCCNTupler
+ * ---------
+ * Summary: The pixel clusters are summed per pixel module per lumi
+ *          lumi section.
+ *
+ * ----------------------------------------------------------------------
+ * Author:  Chris Palmer
+ * ----------------------------------------------------------------------
+ *
+ *
+ ************************************************************/
+
+#include <map>
+
+#include "DataFormats/TrackerRecHit2D/interface/SiPixelRecHitCollection.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "Geometry/TrackerGeometryBuilder/interface/PixelGeomDetUnit.h"
+#include "TrackingTools/TrackAssociator/interface/TrackDetectorAssociator.h"
+#include "HLTrigger/HLTcore/interface/HLTConfigProvider.h"
+#include "SimDataFormats/PileupSummaryInfo/interface/PileupSummaryInfo.h"
+
+#include "TObject.h"
+#include "TH1F.h"
+
+using namespace reco;
+
+class TObject;
+class TTree;
+class TH1D;
+class TFile;
+class RectangularPixelTopology;
+class DetId; 
+
+class PCCNTupler : public edm::EDAnalyzer {
+  public:
+    explicit PCCNTupler(const edm::ParameterSet& ps);
+    virtual ~PCCNTupler();
+    virtual void beginJob();
```

Comment:
I get an error with override on beginJob and endJob so I left those out.  I added it to the rest.  About to commit.

/afs/cern.ch/work/c/capalmer/private/lumi/CMSSW_7_4_0_pre9/src/RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.h:52:18: error: 'virtual void PCCNTupler::beginRun(const edm::Run&, const edm::EventSetup&)' marked override, but does not override
     virtual void beginRun(const edm::Run &, const edm::EventSetup &) override;
                  ^
/afs/cern.ch/work/c/capalmer/private/lumi/CMSSW_7_4_0_pre9/src/RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.h:53:18: error: 'virtual void PCCNTupler::endRun(const edm::Run&, const edm::EventSetup&)' marked override, but does not override
     virtual void endRun(edm::Run const&, edm::EventSetup const&) override;